Задание_8.4/Задание_8.4/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+
+namespace Задание_8._4
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите последовательность чисел");
```

Comment:
Сообщение вводит в заблуждение. По факту вводится натуральное число.
```suggestion
            Console.WriteLine("Введите натуральное число.");
```